### DISCUSSION TITLE
Pokémon Ultra Sun/Moon - Resort ?

### DISCUSSION THREAD

**Discussion by Immersion95** [kwsch/PKHeX#3556] [Q&A] [answered]
Hi,

I have this message but I can't figure out how to solve this :

Invalid: Clone detected (Details).
[main @ [04] (Boîte 4)-28: 368 - Rosabyss - 6CF6DDF57E00.pk7] 368 - Rosabyss - 6CF6DDF57E00.pk7
[main @ Resort-0: 368 - Rosabyss - 6CFBDDF57E00.pk7] 368 - Rosabyss - 6CFBDDF57E00.pk7

Is it possible to access this resort box ?

Best regards.

**Comment by Atrius97** [comment, marked as answer]
The Verify All PKM option cross checks all Pokemon across the save file, including the Daycare, fused Pokemon, and any other documented extra slots.
Sometimes the game does not clear those slots after a Pokemon has been removed, so PKHeX will occasionally be reading partial ghost data from the now empty slot. (there's no Pokemon there, it's just an echo of that Pokemon having been there)
I assume "Resort-0" is referring to the Pokemon having previously visited the Poke Pelago?

**Comment by sora10pls** [comment]
If that is the case, then this should be revised. The Poké Pelago is comprised of nothing _but_ Pokémon that exist in your PC Boxes, as well as potential visitors that you can capture for yourself after some time has passed. Clones are to be expected, since they don’t leave your PC Boxes when they’re in the Poké Pelago.

**Reply by Atrius97** [reply]
This isn't the case for Isle Evelup or Isle Avue, each has 6, 12, or 18 slots (depending on island development tier) for EXP/EV training, and sitting in the hot spring, respectively.
I just checked, and they **_are_** actually removed from your Boxes when selected for those two islands.

The remaining Abeens, Aplenny, and Aphun islands are just occupied by random box pokemon, so they *shouldn't* count as extra slots? I'm unsure about the random wild pokemon that visits Abeens that occasionally will decide to join your team, whether the game generates it right when it appears and counts as a "hidden slot", or if it isn't generated until you choose to take it.
